nipyapi/canvas.py

```diff
@@ -1295,7 +1299,7 @@ def get_controller(identifier, identifier_type='name', bool_response=False):
         if identifier_type == 'id':
             out = handle.get_controller_service(identifier)
         else:
```

Comment:
I'm not sure I understand, we used to call with 0 parameters, no optional, not we always have the reporting optional True, why would it be optional if we want to do it like this?  there is nothing in the context that says include reporting tasks, maybe we just should always?  If that makes sense